Controller/HelperController.php

```diff
@@ -198,14 +203,14 @@ public function getShortObjectDescriptionAction(Request $request)
 
         if ('json' == $request->get('_format')) {
             return new JsonResponse(array('result' => array(
-                'id'    => $admin->id($object),
+                'id' => $admin->id($object),
                 'label' => $admin->toString($object),
             )));
         } elseif ('html' == $request->get('_format')) {
             return new Response($this->twig->render($admin->getTemplate('short_object_description'), array(
-                'admin'           => $admin,
-                'description'     => $admin->toString($object),
-                'object'          => $object,
+                'admin' => $admin,
```

Comment:
hmm.. are you using the latest version of php-cs-fixer? because this change should be reverted...

otherwise you can do this manually